src/pybind/mgr/dashboard/frontend/src/app/shared/components/doc/doc.component.ts

```diff
@@ -10,14 +10,19 @@ import { DocService } from '../../../shared/services/doc.service';
 export class DocComponent implements OnInit {
   @Input() section: string;
   @Input() docText = $localize`documentation`;
+  @Input() noSubscribe: boolean;
 
   docUrl: string;
 
   constructor(private docService: DocService) {}
 
   ngOnInit() {
-    this.docService.subscribeOnce(this.section, (url: string) => {
-      this.docUrl = url;
-    });
+    if (this.noSubscribe) {
```

Comment:
@tspmelo The links where not generating to the login page because in the login page, there is no subscription created. Subscription happen only when someone is logged in and then only the links are generated. Since the links I am using for the login page doesnt require a subscription I've created a noSubscribe @Input() and used that value to see if the incoming cd-doc element require a subscription or not, and based on that the links are generated. I need your opinion on going forward with this approach